Client/Application/AppDelegate.swift

```diff
@@ -484,6 +484,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
                 self.launchFromURL(params)
             }
         }
+
+        guard let extensionDefaults = UserDefaults(suiteName: "group.org.mozilla.ios.Fennec") else { return }
```

Comment:
I don't want this hack in the code. This is a horrible workaround. This also does not work, since it depends on a static group identifier. Instead the group should be taken from `AppInfo.sharedContainerIdentifier`.

There must be a better way to do this.